test/helpers.lua

```diff
@@ -843,6 +843,10 @@ function module.read_nvim_log(logfile, ci_rename)
   return log
 end
 
+function module.mkdir(path)
+  return lfs.mkdir(path)
```

Comment:
the test runner is nvim, do we have `vim.fn` available? (or a TODO once `nvim -ll` becomes `nvim -l`)?

Reply:
> test/helpers.lua:847: attempt to index field 'fn' (a nil value)

I dunno

Reply:
We're currently running the test with `-ll`. Unsure what that means.